src/api/forms/service.js

```diff
@@ -66,11 +70,14 @@ export async function createForm(metadata, author) {
 
   // Create the metadata document
   const { insertedId: _id } = await formMetadata.create(document)
+  const createdForm = mapForm({ ...document, _id })
+  logger.info(`Form ${createdForm.id} created for form "${metadata.title}"`)
 
   // Create the draft form definition
   await draftFormDefinition.create(_id.toString(), definition)
```

Comment:
Then following on from the previous comment, we can update this line:

```suggestion
  await draftFormDefinition.create(metadata.id, definition)
```

Reply:
Done